subcommands.go

```diff
@@ -114,6 +132,46 @@ func (cdr *Commander) ImportantFlag(name string) {
 	cdr.important = append(cdr.important, name)
 }
 
+// VisitGroups visits each command group in lexicographical order, calling
+// fn for each.
+func (cdr *Commander) VisitGroups(fn func(*CommandGroup)) {
+	sort.Sort(byGroupName(cdr.commands))
+	for _, g := range cdr.commands {
+		fn(g)
+	}
+}
+
+// VisitCommands visits each command command in registered order grouped by
```

Comment:
This is confusing, maybe:

VisitCommands visits each command in registered order grouped by command group in lexicographical order, calling fn for each.